index.html

```diff
@@ -0,0 +1,245 @@
+
+                <div class="card3">
+
+                     <h4 class="lectur">Lecture</h4>
+                     <p class="info">Listen to the speakers from various countries about the messages of sharing and opening</p>
+
+                </div>
+
+            </section>
+
+            <section class="individualcard">
+                <div class="card3">
+                     <h4 class="catalog1">CC Exhibition</h4>
+                     <p class="info2">Get to know the new and interesting exhibitions made by our company. </p>
+                 </div>
+            </section>
+
+            <section class="individualcard">
+
+                 <div class="card3">
+
+                     <h4 class="foro">Forum</h4>
+                     <p class="infof">Have the time to share your thoughts and opinions with experts for each topic.</p>
+                 
+                 </div>
+            </section>
+
+            <section class="individualcard">
+                 <div class="card3">
+                     <h4 class="catalog">Workshop</h4>
+                     <p class="info3">Enjoy the lush greenery and good company in a cozy and welcoming enviroment.</p>
+                 
+                 </div>
+            </section>
+
+            <section class="individualcard">
+                 <div class="card3">
+                    <h4 class="ignite">CC Ignite</h4>
+                    <p class="info4">Get opportunities to network with CC affiliates around the world.</p>
+                
+                 </div>
+            </section>
+
+        </section>
+
+        <!--Direction-->
+        <button>
+            <a href="https://goo.gl/maps/TWgiApNouQkwDre69">SEE THE WHOLE PROGRAM</a>
+        </button>
+
+     </section>
+
+     <!--Speakers  Javascript-->
+     <section id="people">
+
+        <h3>Features speakers</h3>
+
+        <hr>
+
+        <div id="person"></div>
+     </section>
+
+     <!--Partners-->
+     <section id="sponsor">
+
+        <h3 class="colaboradores">Sponsors</h3>
+        <hr>
+
+        <section class="sponsors">
+
+            <div class="allcompanys">
+                <img alt="amazon" src="images/amazon.png" class="amazon">
+                <img alt="mozila" src="images/mozilla.png" class="mozilla">
+                <img alt="google" src="images/google.png" class="google">
+                <img alt="ibm" src="images/logoibm1.png" class="ibm">
+            </div>
+
+        </section>
+
+     </section>
+
+     <!--Footer-->
+     <section class="final">
+        
+        <a href="index.html"><img src="./images/company.png"></a>
+
+        <div>
+
+            <p>2023 Maximo Ortelli Rueda | @maximoortelli.</p>
+
+            <p class="bottom">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Consectetur possimus ea, maiores 
+                 facilis culpa tempore quos hic non tempora dolores ipsam inventore at recusandae nostrum fugiat. Ea voluptates autem laudantium.</p>
+
```

Comment:
- [ ] Kindly stick to the design of the background of the mobile. Yours uses that of the desktop instead of a plain one with an arrow key 👍 

### Design:
![image](https://github.com/maximoortelli/Capstone-Proyect/assets/50990925/555602b0-c3b0-46cc-8003-41a522fbedeb)

### Your view:
![image](https://github.com/maximoortelli/Capstone-Proyect/assets/50990925/de707f50-9884-4f57-9100-354ef078b6bd)

Reply:
Thank you very much for the suggestion, the changes have been made satisfactorily 👍😊